src/NinjaTools.FluentMockServer.API/Proxy/ProxyMiddleware.cs

```diff
@@ -12,16 +12,20 @@ public class ProxyMiddleware
         private readonly IOptions<AdminOptions> _adminOptions;
         private readonly RequestDelegate _next;
         private readonly ISetupService _setupService;
+        private readonly ILogService _logService;
 
-        public ProxyMiddleware(RequestDelegate next, ISetupService setupService, IOptions<AdminOptions> adminOptions)
+        public ProxyMiddleware(RequestDelegate next, ISetupService setupService, IOptions<AdminOptions> adminOptions, ILogService logService)
         {
             _next = next;
             _setupService = setupService;
             _adminOptions = adminOptions;
+            _logService = logService;
         }
 
         public async Task InvokeAsync([NotNull] HttpContext context)
         {
+            var path = context.Request.Path;
+            var method = context.Request.Method;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'method'.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4802644)